commons/src/main/java/com/flipkart/flux/commons/dto/Eventsinfo.java

```diff
@@ -0,0 +1,31 @@
+package com.flipkart.flux.commons.dto;
```

Comment:
I was planning to use 'commons' as the one which will be shared to clients. Do you want me to change its name to 'interface' ? 
Sorry, I didn't got what do you mean by api interfaces. ? They are just DTOs which clients should be aware of as well. Can you please explain more ?